lib/cli.js

```diff
@@ -6,7 +6,11 @@ const path = require('path');
 const fs = require('fs-extra');
 const { settings } = require('./settings');
 
-const { getTestSuitePaths, distributeTestsByWeight } = require('./test-suites');
+const {
```

Comment:
if we want to have `easy` keeping up-to-date with parent then it's better not to restyle upstream code